extensions/vscode/.vscode/launch.json

```diff
@@ -6,12 +6,24 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Run Extension",
+      "name": "Debug with external API agent on 9001",
       "type": "extensionHost",
       "request": "launch",
       "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
       "outFiles": ["${workspaceFolder}/dist/**/*.js"],
-      "preLaunchTask": "${defaultBuildTask}"
+      "preLaunchTask": "${defaultBuildTask}",
+      "env": { "POSIT_PUBLISHER_USE_EXTERNAL_AGENT": "TRUE" },
+      "cwd": "${workspaceFolder}/../../test/sample-content"
```

Comment:
Are the `cwd` attributes important here? We didn't have it previously with the default Debug.

Reply:
Yes, they are. You have to launch the agent within the same CWD as the extension has as an open workspace, otherwise, all paths go out of sync.